packages/telemetry-node/package.json

```
@@ -0,0 +1,35 @@
+{
+  "name": "@replayio/telemetry-node",
+  "version": "1.0.0",
+  "description": "Node utilities for telemetry",
```

Feel free to tweak this anyhow you want, we probably could even just omit the description. Either way is fine with me
```suggestion
  "description": "Node utilities for Replay.io's own telemetry",
```